### PR TITLE
Parse tuples in the expression

### PR DESCRIPTION
**Pull request by Thirumalai-Shaktivel**
Closes: #719

### CODE REVIEW

**Comment by Thirumalai-Shaktivel** [comment]
This PR is ready!

File: src/lpython/parser/parser.yy [hunk 1]
```diff
@@ -281,8 +282,8 @@ void yyerror(YYLTYPE *yyloc, LFortran::Parser &p, const std::string &msg)
 %left "%" "//" "/" "@" "*"
 %precedence UNARY
 %right "**"
-%precedence AWAIT
-%precedence YIELD
+/* %precedence AWAIT */
+/* %precedence YIELD */
```
Comment (certik) [inline src/lpython/parser/parser.yy:286]: If this is not needed, then you can remove it?

**Review by certik — APPROVED** [review]
That looks good to me.

**Comment by Thirumalai-Shaktivel** [comment]
Thanks for the review!

**Pull request merged**